src/components/change-requests/change-request-details/change-request-details/change-request-details.tsx

```diff
@@ -55,26 +54,24 @@ interface ChangeRequestDetailsProps {
 const ChangeRequestDetails: React.FC<ChangeRequestDetailsProps> = ({
   changeRequest
 }: ChangeRequestDetailsProps) => {
-  const reviewBtns = (
-    <div className={styles.btnsContainer}>
-      <ActionButton
-        link={`/change-requests/${changeRequest.crId}/accept`}
-        icon={faThumbsUp}
-        text="Accept"
-      />
-      <ActionButton
-        link={`/change-requests/${changeRequest.crId}/deny`}
-        icon={faThumbsDown}
-        text="Deny"
-      />
-    </div>
+  const reviewDropdown = (
+    <DropdownButton id="review-dropdown" title="Review">
+      <Dropdown.Item as={Link} to={`/change-requests/${changeRequest.crId}/accept`}>Accept</Dropdown.Item>
```

Comment:
update them and then use shared routes instead of hard-coded strings https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/092644cd1ebac0f9fcd70b458d8d5fa832409a08/src/shared/routes.ts#L20